### ISSUE TITLE
Improvements for wasteland outskirts

### ISSUE THREAD

**Issue by Torbenbeta** [Axema/Vampiric-Stuff#33]
### Description of problem
While playing the mod, I stumbled upon the wasteland outskirts. I noticed several things, which could be improved.

### Suggestions

- The items in the fort are not owned by the faction, so that they can be stolen. (cf. the items in the evacuation centre, which belong to the free merchants.
- The gun stall owner does not sell guns in contradiction to his owns words . Either add this to his inventory or change description (could be an artefact by the rng though).
- The chat options of the guards are lacking. Especially a description of what this place is seems appropriate. 
- The quest to search for arcane artefacts does not appear in the quest window (I suppose this is not intended, else ignore this point).

### Additional context
I only visited this location other locations could benefit from point 1 as well.

**Comment by Axema** [comment]
Yeah, the outskirt was left incomplete, and outdated for a long time.

* On your 1st suggestion, factions owning items in respected area wasn't possible, but now it's doable.
* 2nd, it's probably because the merchant can't load its shopkeep items. I will try to come up with a different way for trading.
* 3rd. The guards only tells you more about their place (and further plans) when you've done some quests.
* Lastly, the arcane seeker checks your inventory for available artefacts. It's more like trading, actually.
